modules/sitestudio_config_management/sitestudio_config_management.install

```diff
@@ -19,9 +19,18 @@ function sitestudio_config_management_install($is_syncing) {
     $updated_ignore_config = array_unique(array_merge($existing_ignore_config, $new_ignore_config));
     $config->set('ignored_config_entities', $updated_ignore_config);
     $config->save(TRUE);
-    if (!\Drupal::config('cohesion.sync.settings')->get('enabled_entity_types')) {
-      _sitestudio_config_management_import_configuration('cohesion.sync.settings', 'sitestudio_config_management');
+
+    // Merge active configuration with new config.
+    $new_cohesion_sync_config = _sitestudio_config_management_import_configuration('cohesion.sync.settings', 'sitestudio_config_management');
+    $active_cohesion_sync = \Drupal::configFactory()->getEditable('cohesion.sync.settings');
```

Comment:
this line should be the first line, below code actually write to the active config.
`$new_cohesion_sync_config = _sitestudio_config_management_import_configuration('cohesion.sync.settings', 'sitestudio_config_management');`

Reply:
@rajeshreeputra yes we need to overwrite enabled_entity_types config other configs get overwritten only in case of empty value.